src/work-allocation/release2/components/task-field/task-field.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
```

Comment:
Same as the template. I can see no reason at all for this to have been duplicated.